modules/govuk_search/manifests/gor.pp

```diff
@@ -0,0 +1,55 @@
+# == Class: govuk_search::gor
+#
+# Dump POST traffic to file so that it can be replayed. This means we have a simple
+# way to recover from failure as we can restore the last backup and then replay the
+# traffic (POST requests are for data insert/update only)
+#
+# Traffic can then be replayed by follow the guide at:
+# https://github.com/buger/goreplay/wiki/Capturing-and-replaying-traffic
+#
+# === Parameters
+#
+# [*enabled*]
+#   Boolean to determine if Search traffic will be saved; defaults to false.
+#
+# [*output_path*]
+#   File location for files to be saved to.
+#
+class govuk_search::gor (
+  $output_path = undef,
```

Comment:
I don't think there's harm in setting a sane default.

Reply:
Set this to `/var/log/gor_dump`